paper-tabs.html

```diff
@@ -133,15 +133,19 @@
         -moz-flex-basis: auto;
         -ms-flex-basis: auto;
         flex-basis: auto;
+        @apply(--layout-horizontal);
       }
 
       #tabsContent.scrollable {
         position: absolute;
+        min-width: 100%;
         white-space: nowrap;
       }
 
-      #tabsContent.horizontal {
-        @apply(--layout-horizontal);
+      #tabsContent.scrollable > ::content > * {
+        -ms-flex-shrink: 0;
+        -webkit-flex-shrink: 0;
+        flex-shrink: 0;
```

Comment:
It works fine without it in Chrome, but it is needed for IE10 (IIRC IE11 as well, can't remember if it was needed for Edge). Could just make it `-ms-flex-shrink`.